libethcore/Miner.h

```diff
@@ -123,10 +120,9 @@ inline std::ostream& operator<<(std::ostream& _out, WorkingProgress _p)
 			}
 		}
 
-		_out << "gpu/" << i << " " << EthTeal << std::fixed << std::setw(5) << std::setprecision(2) << mh << EthReset;
+		_out << " gpu/" << i << " " << EthTeal << std::fixed << std::setprecision(2) << mh << EthReset;
```

Comment:
Someone suggested to numerate GPUs as `gpu0` not `gpu/0`. I don't mind.

Reply:
Done